website/content/blog/announcing-the-platform-engineering-maturity-model.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Announcing the Platform Engineering Maturity Model'
+date:   2023-11-01 00:00:00 +0000
+author: Abby Bangser, Josh Gavant
+---
+
+The CNCF Platforms Working Group (WG) is excited to present the first release of a platform engineering maturity model which provides a more concrete application of the extremely well received white paper from this past April.
+
+We want to thank the almost 50 people who have contributed their time and ideas to make this model reflect the state of companies both small and large and across both fast moving and highly regulated industries. Download now as a [PDF](https://github.com/cncf/tag-app-delivery/raw/main/platforms-maturity-model/v1/assets/platform-eng-maturity-model-v1.0.pdf) or view it on our [website](https://appdelivery.cncf.io/whitepapers/platform-eng-maturity-model/).
```

Comment:
Perhaps swap the order of these sentences and put the "Download it now" sentence in its own paragraph?
Could also end the "thanks" sentence with "; all of them are listed below."